cpp/oneapi/dal/algo/decision_forest/infer_types.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
-* Copyright 2020 Intel Corporation
```

Comment:
is it necessary change, due to this file was created before?

Reply:
Thanks for mentioning. Fixed this.